tests/ci/build_run_benchmarks.sh

```diff
@@ -7,49 +7,46 @@ set -x
 # set default value of directory name
 if [ -z "${PR_FOLDER_NAME}" ]; then export PR_FOLDER_NAME=aws-lc; fi
 
+# Get AWS_ACCOUNT_ID
+if [ -z "${AWS_ACCOUNT_ID}" ]; then AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text); fi
+
+AWSLC_PR_ROOT=$(pwd)
+
+cd ..
+
 # run this from the bm_framework root directory!
-OPENSSL_ROOT=$(pwd)/openssl
-BORINGSSL_ROOT=$(pwd)/boringssl
 AWSLC_PR_ROOT=$(pwd)/"${PR_FOLDER_NAME}"
 AWSLC_PROD_ROOT=$(pwd)/aws-lc-prod
 
+NUM_CPU_THREADS=$(nproc)
+
 # clone the various repositories we need (we already have aws-lc-pr since we need it to run this script)
 git clone https://github.com/awslabs/aws-lc.git aws-lc-prod
-git clone https://boringssl.googlesource.com/boringssl
-git clone --branch OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git
-
-# build OpenSSL
-mkdir openssl/build
-(cd openssl && ./config --prefix="${OPENSSL_ROOT}"/build --openssldir="${OPENSSL_ROOT}"/build)
-make -C openssl
-make install -C openssl
-
-# build BoringSSL
-mkdir boringssl/build
-cmake -Bboringssl/build -Hboringssl -GNinja -DCMAKE_BUILD_TYPE=Release
-ninja -C boringssl/build
 
 # build AWSLC pr
-mkdir "${PR_FOLDER_NAME}"/build
+mkdir -p "${PR_FOLDER_NAME}"/build
+mkdir -p "${PR_FOLDER_NAME}"/install
 cmake -B"${PR_FOLDER_NAME}"/build -H"${PR_FOLDER_NAME}" -GNinja -DCMAKE_BUILD_TYPE=Release \
-  -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}" \
-  -DBORINGSSL_INSTALL_DIR="${BORINGSSL_ROOT}" \
-  -DOPENSSL_INSTALL_DIR="${OPENSSL_ROOT}"
+  -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}"/install \
+  -DCMAKE_INSTALL_PREFIX="${AWSLC_PR_ROOT}"/install
```

Comment:
To speed up all these build you could turn off testing and libssl.

Reply:
Turning off LIBSSL causes it to not build the awslc_bm tool